fenix/app/nimbus.fml.yaml

```diff
@@ -270,6 +270,14 @@ features:
         type: Boolean
         default: true
 
+  extensions-GA:
+    description: Features for web extension general availability.
+    variables:
+      process-enabled:
+        description: If the web extension process should be enabled or not.
+        type: Boolean
+        default: false
```

Comment:
Idle question— should this be in `gecko.fml.yaml`? Thinking about it, this is Fenix code, so I can see it here.

However, I think naming the feature `extensions` or `gecko-view` may be a better path forward.

Reply:
Thanks for the head ups! I think it could be in any file those files, I noticed the [gecko.fml.yaml](https://github.com/mozilla-mobile/firefox-android/blob/c50704866dea03f634342732f8bd3654aaa4f55b/android-components/components/browser/engine-gecko/geckoview.fml.yaml#L10-L11) is targeting   `debug` and `release`, I prefer to keep the changes here for now.